syntax_and_semantics/blocks_and_procs.md

```diff
@@ -198,6 +198,18 @@ method &.+(2)
 method &.[index]
 ```
 
+If the block is not the only parameter being passed to the method, the short syntax argument must be supplied within the argument list.
```

Comment:
```suggestion
If the block is not the only parameter being passed to the method, the short syntax argument can be supplied within the argument list.
```

Reply:
Why not `must` here?  Since it was _wasn't_ supplied in the argument list it wouldn't error, like the issue https://github.com/crystal-lang/crystal/issues/7549

Reply:
I reconsidered and think this sentence and line 166 need to be changed entirely. The short syntax argument is *always* supplied in the argument list. If there are no other arguments, it's the *only* element in that list, but it's always a normal argument. This should be clarified.
Given `foo &.bar` and `foo { |f| f.bar }`, it looks like `&.bar` is syntactically equivalent to `{ |f| f.bar }`, but in fact it's not. `&.bar` is a regular argument while `{ |f| f.bar }` is a block. This is obvious when adding the optional parenthesis: `foo(&.bar)` vs. `foo() { |f| f.bar }`